auto_dev/data/templates/protocols/protodantic.jinja

```diff
@@ -16,3 +16,6 @@ from {{ primitives_import_path }} import (
 MAX_PROTO_SIZE = 2 * 1024 * 1024 * 1024  # 2 GiB in bytes
 
 {{ formatter.render(file) }}
+
+for cls in BaseModel.__subclasses__():
+    cls.update_forward_refs()
```

Comment:
Here the magic happens! This makes all the scaffolded pydantic models from the .proto files compatible with hypothesis library. This prevents the need for any complex scaffolding logic for the tests and instead to simply use `st.from_type()`!